.circleci/helm-values/staging.proxy-scram.yaml

```diff
@@ -6,7 +6,7 @@ image:
 
 settings:
   authBackend: "console"
-  authEndpoint: "https://console.stage.neon.tech:9095/management/api/v2"
+  authEndpoint: "http://console-staging.local/management/api/v2/healthz"
```

Comment:
Hm, why `/healthz`? This string would be a base to construct API calls like `/management/api/v2/proxy_get_scram_secret?role=aaa&cluster=bbb`. Also port is 9095 as per `https://github.com/neondatabase/cloud/blob/main/helm/zenith-console/values-stage.yaml#L78-L85`.